GitSearch/Application/SearchResult/SearchResultCell.swift

```diff
@@ -112,7 +34,93 @@ class SearchResultCell: UICollectionViewCell, UICollectionViewDelegateFlowLayout
         return layoutAttributes
     }
     
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupUI()
+    }
+    
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 }
+
+extension SearchResultCell {
+
+    func fillInfo(for item: SearchItem, isLast: Bool) {
+        self.lastCell = isLast
+        self.title.text = item.name
+        self.author.text = "by \(item.owner?.login ?? "")"
+        self.subTitle.text = """
+        \(item.description ?? "")
+        """
```

Comment:
No need in multiline string literal here:
```suggestion
        self.subTitle.text = item.description
```

Reply:
vestige of past approac, will fix

Reply:
[`UILabel.text`](https://developer.apple.com/documentation/uikit/uilabel/1620538-text) property has type of `String?`, so it can handle nil.